README.rst

```diff
@@ -749,13 +749,13 @@ setting ``REDIS_CLIENT_KWARGS``.
 Closing Connections
 ~~~~~~~~~~~~~~~~~~~
 
-django-redis do not closes all the connections to Redis server on close().
+django-redis doesn't close all the connections to Redis server on close().
 
-You can change this default behaviour for all caches by setting ``DJANGO_REDIS_CLOSE_CONNECTION = True``
-in the django settings or at cache level by setting ``CLOSE_CONNECTION: True`` in the ``OPTIONS``
+You can change this default behaviour for all caches by the ``DJANGO_REDIS_CLOSE_CONNECTION = True``
+in the django settings (globally) or (at cache level) by setting ``CLOSE_CONNECTION: True`` in the ``OPTIONS``
 for each configured cache.
 
-Setting True as a value will instruct the DefaultClient to close all the connections, irrespectively of its current usage.
+Setting True as a value will instruct the DefaultClient to close all the connections on close(), irrespectively of its current usage.
```

Comment:
done
```
Setting True as a value will instruct the django-redis to close all the connections (since v. 4.12.2), irrespectively of its current usage.

```